man/rmd/indexed-error.Rmd

```diff
@@ -0,0 +1,121 @@
+The `purrr_error_indexed` class is thrown by [map()], [map2()], [pmap()], and friends.
+It wraps errors thrown by applying `.f` to individual elements giving additional information about the location of the error.
+
+# Structure
+
+`purrr_error_indexed` has two fields:
+
+-   `location`: the location of the error as a single integer.
+-   `name`: the name of the location as a string. If the element was not named, `name` will be `NULL`
+
+`purrr_error_indexed` inherits from `rlang_error` which means that it also has a `parent` field which contains the original error thrown by `.f`.
```

Comment:
rlang errors do not necessarily have a `parent` field. (Also technically base errors might contain one if they want to implement the chained error protocol of rlang.). Maybe:

> `purrr_error_indexed` is always constructed as a child error (see the `parent` argument of [rlang::abort()]). It has a `parent` field that contains the original error thrown by `.f`.